tests/Serializer/JsonSerializationTest.php

```diff
@@ -109,6 +109,13 @@ protected function getContent($key)
             $outputs['maxdepth_skippabe_object'] = '{"a":{"xxx":"yyy"}}';
             $outputs['array_objects_nullable'] = '[]';
             $outputs['type_casting'] = '{"as_string":"8"}';
+            $outputs['authors_inline'] = '{"as_string":"8"}';
```

Comment:
this is not correct, see https://github.com/schmittjoh/serializer/pull/974/files#r202676680